lib/utils/createModifier.js

```diff
@@ -41,6 +34,5 @@ function createModifier({ modelClass, modifier, modifiers, args }) {
 }
 
 module.exports = {
-  createModifier,
-  ModifierNotFoundError
+  createModifier
```

Comment:
@koskimas I'm not sure if you want to keep exporting this as a named export now that there's only one export in the file?